SinaiStCatherine/SinaiNewEt001.xml

```diff
@@ -218,34 +261,144 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        in part possibly a transcription of a phrase in a non-Ethiopian language.</desc>
+                      <q xml:lang="gez">በሥሙለግዚዕነ፡ ኢየሱሥ፡ ክርስቶስ፡ ፃሐፍኩ፡ ዐነግርማኖስ፡ ዘሪኢኩዕምክርሥቲያን፡ ወይብልቀሢሥ፡ ዕዘይ፡ ወጥን፡ ወይብል፡ ዕብለይጦስ፡
+                        <gap reason="lost" unit="chars" quantity="7"/>ወተዎስ፡ ባዶቴኒ፡ ኒኬያይኬይ፡ <gap reason="lost" unit="chars" quantity="3"/>ስ፡ ቶስየዎን፡ ዐስቶን፡ የወን፡ ዐሜን። ወይብል፡ 
+                        ኩለዕለውስተቤተኽርስቲያን፡ አሜን</q>          
+                    </item>
+                    
+                    
+                    <item xml:id="e1">
+                      <locus from="3v"/>
+                      <desc type="Unclear">A blessing note in the upper margin, partly ununderstandable.</desc>
+                      <q xml:lang="gez">ይባረክ፡ ወይሱን፡</q>                        
+                    </item>
+                    
+                    <item xml:id="e2">
+                      <locus from="4r"/>
+                      <desc type="Gloss">A translation of one word into an Ethio-Semitic language.</desc>
+                      <q xml:lang="gez">ገቦ፡ ብህል፡ መሰር፡</q>                        
+                    </item>
+                    
+                    <item xml:id="e3">
+                      <locus target="#3ra #4ra #6vb #13va #10ra #14va #15r #22va #23va "/>
+                      <desc type="Correction">Minor interlinear corrections.</desc>                        
+                    </item>
+                  </list>
+                </additions>
                   
-            <!--    <history>
+                  <bindingDesc>
+                    <binding xml:id="binding" contemporary="false">
+                   <decoNote xml:id="b7" type="SewingStations">The quires of <ref target="#p2">Unit 2</ref> were originally sewn on 
+                        two pairs of sewing station (see, e.g., the centre fold between <locus target="#13v"/> and <locus target="#14r"/>).
+                   </decoNote>             
+                      
+                    </binding>
+                  </bindingDesc>
+                </physDesc>
+                
+            <history>
                   <origin>
-                    <origDate notBefore="" notAfter="" evidence="lettering"/>
+                    <origDate notBefore="1350" notAfter="1420" evidence="lettering">Second half of the 13th-/early 14th-century</origDate>
+                    The manuscript was commissioned by <persName ref="PRS14170Tadewos"/>.
                   </origin>
-                </history>-->
+                </history>
               </msPart>
+              
+              <msPart xml:id="p3">
+                <msIdentifier/>
+                
+                <msContents>
+                  <summary/>
+              
+              <msItem xml:id="p1_i3">
+                <locus from="26r" to="27v"/>
+                <title type="incomplete" ref="LIT2000Mazmur">Fol. 26r: Ps 35፡30 (from ወይልበሱ፡ ኃፍረተ፡) to the end, and Ps 46:1-9.1. 
```

Comment:
I think this is too much content for the title element. Maybe the details would be better in a note (then also marking up the Geez text with foreign)

Reply:
ok